src/components/AutoSaveIndicator.tsx

```diff
@@ -0,0 +1,34 @@
+import { AutoSaveStatus, useAutoSaveStatus } from "@homebound/form-state";
+import { assertNever } from "@homebound/form-state/dist/utils";
```

Comment:
Hm, do we have an `assertNever` here locally in Beam? Would be nice to avoid the dist import.

Reply:
We do not. And this is the only one intellisense is picking up.

Reply:
Can we add one in to Beam, please?